lib/publiccloud/acr.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
```

Comment:
based on the new rules https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/6af8aeda0709bb263826c0224c0368a6b750b883#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R49 year should be omitted

Reply:
done